src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1838,6 +1845,11 @@ export function showWaterfallPlot(horzSelection:AxisMenuSelection, vertSelection
     || (horzSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY && vertSelection.dataType === NONE_SELECTED_OPTION_STRING_VALUE);
 }
 
+export function showPlotWithClinicalAttrsAndTreatments(horzSelection:AxisMenuSelection, vertSelection:AxisMenuSelection):boolean {
```

Comment:
you can call this function in the `@computed` in `PlotsTab` with the same logic, so as not to duplicate logic